### PR TITLE
Changes to step 3 errors

### LINKED ISSUE #1266: Visually tweak Step 3 Errors design implementation
via live bug squash!

### PR DESCRIPTION
**Pull request by hbillings**
Closes #1266 and #1317.

### CODE REVIEW

**Comment by hbillings** [comment]
Looking thusly:
screencapture-0-0-0-0-8000-data-capture-step-3-errors-1487805996974

**Comment by jseppi** [comment]
Is this ready for review @hbillings?

**Comment by hbillings** [comment]
@jseppi 'tis!

File: frontend/source/sass/components/_alerts.scss [hunk 1]
```diff
@@ -43,8 +43,17 @@ alerts-widget:focus {
...
+.message-error h3,
+.message-error p {
   color: $color-secondary-dark;
+  font-weight: $font-weight-bold;
+}
+
+.message-error h3:before {
+  width: $h3-font-size;
+  height: $h3-font-size;
+  content: url('../../images/icon-warning.svg');
```
Comment (jseppi) [inline frontend/source/sass/components/_alerts.scss]: I can't seem to find `icon-warning.svg`. Forgot to commit it?

Comment (jseppi) [inline frontend/source/sass/components/_alerts.scss]: screen shot 2017-02-24 at 4 49 38 pm

Comment (hbillings) [inline frontend/source/sass/components/_alerts.scss]: Oh, _weird_, it's not in my local repo any more either. 0_0 Will re-add!

**Pull request merged**